src/components/GoalsComponents/GoalConfigModal/ConfigGoal.tsx

```diff
@@ -269,9 +269,7 @@ const ConfigGoal = ({ goal, action }: { action: "Update" | "Create"; goal: GoalI
                   body: (
                     <div>
                       <div>
-                        <span>
-                          {perDayHrs[0] === perDayHrs[1] ? perDayHrs[0] : `${perDayHrs[0]} - ${perDayHrs[1]}`} hrs / day
-                        </span>
+                        <span>{timeDiff} hrs / day</span>
```

Comment:
hi @webAbhi can you please revert to old code. because now if i move slider of hrs/day and hrs/week i can't see the change. 
![image](https://github.com/tijlleenders/ZinZen/assets/39093015/5f29844f-26d8-415e-afb8-0e6967c1b2e1)